src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -6,6 +6,8 @@
  * a messaging session.
  */
 /* eslint  @typescript-eslint/no-explicit-any: 0, @typescript-eslint/explicit-module-boundary-types: 0 */
+import PrefetchOn from './PrefetchOn';
```

Comment:
This import should be above the line 4 where the MessagingSessionConfiguration docstring starts. Check this file for more on the format:
https://github.com/aws/amazon-chime-sdk-js/blob/main/src/meetingsession/MeetingSessionConfiguration.ts#L4-L10